set.mm

```diff
@@ -458949,8 +458949,8 @@ Positive operators (cont.)
     pjima.1 $e |- A e. SH $.
     pjima.2 $e |- B e. CH $.
     $( The image of a projection.  Lemma 5 in Daniel Lehmann, "A presentation
-       of Quantum Logic based on an _and then_ connective"
-       ~ http://www.arxiv.org/pdf/quant-ph/0701113 p. 20.  (Contributed by NM,
+       of Quantum Logic based on an _and then_ connective",
+       ~ https://doi.org/10.48550/arXiv.quant-ph/0701113 .  (Contributed by NM,
```

Comment:
deletes page 20?

Reply:
Yes. It's more standard to cite that way ("Lemma 5 of [xxx]") and the page is unnecessary (it is also less stable than the lemma number, now that a given paper can be available in various formats).

Reply:
Oh, I didn't know arxiv had that issue too. It is very much an issue with the HoTT book, which has been available in different formats (with different page numbers, but the same theorem numbers) from the start.

Reply:
Well, maybe I wasn't precise enough: if you select a given version of the arXiv preprint and a given format, say PDF, then the page does not change. But you now you have html versions a newer preprints, where page numbers make no sense, and you may also land via internet search on the journal version, which will most probably have different page numbers, while the lemma/proposition/theorem number is more likely (if the author did put the corrected version on the arXiv) to be the same.
Anyway, these are minor details.

Reply:
Ah got it, somewhat similar results but due to different mechanisms (in the HoTT Book case, the scripts literally generate multiple formats - PDFs with different page sizes - from the same LaTeX source).